schema_based_on_diagram.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE clinic;
+CREATE TABLE patients (id INT PRIMARY KEY, name varchar(50), date_of_bith date);
+CREATE TABLE medical_histories (id INT PRIMARY KEY,admitted_at TIMESTAMP, patient_id INT,status VAR
+CHAR(20));
+CREATE TABLE treatments (id INT PRIMARY KEY,type VARCHAR(25),name VARCHAR(50));
+CREATE TABLE invoice_items (id INT PRIMARY KEY, unit_price DECIMAL,quantity INT,total_price DECIMAL,
+invoice_id INT,treatment_id INT);
+CREATE TABLE invoices (id INT PRIMARY KEY, total_amount DECIMAL, generated_at TIMESTAMP,payed_at TI
+MESTAMP,medical_history_id INT);
+
```

Comment:
Great Job so @andym80 and @Aseadd 

- [ ] Please all "join table" for the many-to-many relationships created, kindly create a table between medical_histories and treatments.
```sql
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```